packages/slice-machine/lib/env/manifest.ts

```diff
@@ -49,63 +55,6 @@ export function extractRepo(parsedRepo: ParseResult): string | undefined {
   }
 }
 
-function validateEndpoint(endpoint: string, parsedRepo: ParseResult): boolean {
-  try {
-    switch (parsedRepo.type) {
-      case ParseResultType.Listed: {
-        if (!parsedRepo?.subDomains?.length) {
-          return false;
-        }
-        if (!endpoint.endsWith("api/v2") && !endpoint.endsWith("api/v2/")) {
-          return false;
-        }
-        return true;
-      }
-      default: {
-        return false;
-      }
-    }
-  } catch (e) {
-    const message =
-      "[api/env]: Unrecoverable error. Could not parse api endpoint. Exiting..";
-    console.error(message);
-    return false;
-  }
-}
-
-function validate(manifest: Models.Manifest): ManifestInfo {
-  if (!manifest.apiEndpoint) {
-    return {
-      state: ManifestState.MissingEndpoint,
-      message: Messages[ManifestState.MissingEndpoint],
-      content: null,
-    };
-  }
-  const endpoint = fromUrl(manifest.apiEndpoint);
-  const parsedRepo = parseDomain(endpoint);
-  if (!validateEndpoint(manifest.apiEndpoint, parsedRepo)) {
-    return {
-      state: ManifestState.InvalidEndpoint,
-      message: Messages[ManifestState.InvalidEndpoint],
-      content: null,
-    };
-  }
-  const repo = extractRepo(parsedRepo);
-  if (!repo) {
-    return {
-      state: ManifestState.InvalidEndpoint,
-      message: Messages[ManifestState.InvalidEndpoint],
-      content: null,
-    };
-  }
-  return {
-    state: ManifestState.Valid,
-    message: Messages[ManifestState.Valid],
-    content: manifest,
-    repo,
-  };
-}
-
 function handleManifest(cwd: string): ManifestInfo {
```

Comment:
I think this could be simplified.
We have functions in the core to retrieve the manifest.